### PR TITLE
debugpb: init metrics request

### PR DESCRIPTION
**Pull request by AndreMouche**
@overvenus @hicqu  PTAL

### CODE REVIEW

File: proto/debugpb.proto [hunk 1]
```diff
@@ -148,3 +151,10 @@ message ListFailPointsResponse {
 
     repeated Entry entries = 1;
 }
+
+message GetMetricsRequest {
+}
+
+message GetMetricsResponse {
+   string metrics = 1;
```
Comment (overvenus) [inline proto/debugpb.proto]: How about name it as `tikv`? We may dump rocksdb and gRPC metrics in the future.

File: proto/debugpb.proto [hunk 2]
```diff
@@ -148,3 +151,10 @@ message ListFailPointsResponse {
 
     repeated Entry entries = 1;
 }
+
+message GetMetricsRequest {
+}
```
Comment (siddontang) [inline proto/debugpb.proto]: should add a content-type to return text or protobuf format data.

File: proto/debugpb.proto (hunk 1 above)
Comment (siddontang) [reply proto/debugpb.proto]: Commonly, we treat metrics as Prometheus metrics especially. Like Go, we use `/metrics` for the Prometheus handler. 

Maybe for others like RocksDB and gRPC, we can use another status or statistics API.

File: proto/debugpb.proto (hunk 2 above)
Comment (overvenus) [reply proto/debugpb.proto]: But the response is a string.

Comment (AndreMouche) [reply proto/debugpb.proto]: Could we support text first？

**Comment by overvenus** [comment]
A friendly ping.

**Pull request merged**